src/main/java/com/powsybl/openloadflow/network/impl/LfHvdcImpl.java

```diff
@@ -40,16 +42,33 @@ public class LfHvdcImpl extends AbstractElement implements LfHvdc {
 
     private boolean acEmulation;
 
-    public LfHvdcImpl(String id, LfBus bus1, LfBus bus2, LfNetwork network, HvdcAngleDroopActivePowerControl control,
+    private final double pMaxFromCS1toCS2;
+
+    private final double pMaxFromCS2toCS1;
+
+    public LfHvdcImpl(String id,
```

Comment:
Just a detail, can you keep the format used in the rest of the code? We put in line. It is just to have an homogeneous way to code.

Reply:
Done. But I do not like this layout for long list of arguments (larger than a laptop screen).
It's OK here.